source/tool/chipsec/hal/acpi_tables.py

```diff
@@ -584,3 +584,27 @@ def __str__( self ):
 ACPI Table Entries:
 %s
 """ %( ''.join( ['0x%016X\n' % addr for addr in self.Entries]))
+
+########################################################################################################
+#
+# FACP Table
+#
+########################################################################################################
+
+class FACP (ACPI_TABLE):
+  def __init__( self ):
+    self.DSDT = None
+    self.X_DSDT = None
+
+  def parse( self, table_content ):
+    self.DSDT = struct.unpack('=I', table_content[4 : 8])

```

Comment:
XSDT [uses '='](https://github.com/raisfathin/chipsec/blob/2cbed174fbf7e37555e67e195f7af358e893ae0f/source/tool/chipsec/hal/acpi_tables.py#L554) though to parse its entries. Is there any reason why it should be '<'?